src/lib.rs

```diff
@@ -65,15 +68,16 @@ impl<T> Default for OneOrMany<T> {
 }
 
 /// JWT Claims.
+#[serde_as]
```

Comment:
If we are using serde_with, we might as well use [`skip_serializing_none`](https://docs.rs/serde_with/3.2.0/serde_with/attr.skip_serializing_none.html).